pkgs/top-level/all-packages.nix

```diff
@@ -5192,6 +5192,7 @@ with pkgs;
   cudaPackages_11_4 = callPackage ./cuda-packages.nix { cudaVersion = "11.4"; };
   cudaPackages_11_5 = callPackage ./cuda-packages.nix { cudaVersion = "11.5"; };
   cudaPackages_11_6 = callPackage ./cuda-packages.nix { cudaVersion = "11.6"; };
+  cudaPackages_11_7 = callPackage ./cuda-packages.nix { cudaVersion = "11.7"; };
   cudaPackages_11 = cudaPackages_11_6;
```

Comment:
Is it really not possible to bump `cudatoolkit` and `cudatoolkit_11` without cuda-samples?

Reply:
I've bumped it.